src/h5web/visualizations/heatmap/IndexAxis.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Axis } from '@vx/axis';
+import { format } from 'd3-format';
+import styles from './HeatmapVis.module.css';
+import { IndexAxisProps, getOrientedAxisProps } from './utils';
+
+function IndexAxis(props: IndexAxisProps): JSX.Element {
+  const commonTickLabelProps = {
+    className: styles.tick,
+    textAnchor: 'inherit' as const,
+    dominantBaseline: 'inherit' as const,
```

Comment:
Text alignement props are inherited as they are set separately for `bottom` and `left` orientation. After thinking about it, I think it is clearer to move everything in the `getOrientedAxisProps`.

Reply:
You can't move these to CSS ?

Reply:
:man_facepalming: